### ISSUE TITLE
Three-finger swipe with VoiceOver enabled does not work on episode page

### ISSUE THREAD

**Issue by amugofjava** [amugofjava/anytime_podcast_player#145]
**Describe the bug**
Swiping with three fingers is the accessible way of scrolling through a list with VoiceOver/TalkBack; however, this gesture does not work within the podcast details page.

**To Reproduce**
1. Enable VoiceOver/TalkBack.
2. Open a podcast from the library or search results.
3. Attempt to scroll through the list of episodes using a three-finger swipe.
4. The list does not scroll.

**Expected behavior**
The episode list should scroll using this gesture as it does within other scrollable lists within the application.

**Smartphone (please complete the following information):**
 - iPhone SE, iOS 18.

**Additional context**
The issue here is likely caused by the custom scroll view and slivers used to build the screen. There are two scroll actions within the episode page and the accessible gesture is being lost.
